src/index.html

```diff
@@ -2,13 +2,103 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <meta
+    name="viewport"
+    content="
+      width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0
+    "
+  >
+  <meta
+    http-equiv="X-UA-Compatible"
+    content="ie=edge"
+  >
   <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
+  <link
+    rel="stylesheet"
+    href="./style.css"
+  >
 </head>
 <body>
-  <h1>Moyo header</h1>
+  <header class="bigger-nav">
+    <div class="logo">
+      <img
+        src="images/logo.png"
+        alt="logo"
+      >
+    </div>
+    <nav >
+      <ul class="menu">
+        <li class="menu-item">
+          <a
+            href="#"
+            class="item-link is-active"
+          >
+            apple
```

Comment:
```suggestion
            Apple
```
in case you need to remove uppercase and use usual styles